api/api/controllers/kvdbs_controller.py

```diff
@@ -0,0 +1,206 @@
+from connexion.lifecycle import ConnexionResponse
+
+from api.controllers.util import json_response
+from api.util import remove_nones_to_dict, parse_api_param, raise_if_exc
+from wazuh import kvdbs
+from wazuh.core.cluster.dapi.dapi import DistributedAPI
+
+logger = logging.getLogger('wazuh-api')
+
+
+async def get_kvdbs(pretty: bool = False, wait_for_complete: bool = False,
+                    offset: int = 0, limit: int = None, select: list = None,
+                    sort: str = None, search: str = None, q: str = None,
+                    distinct: bool = False, type_: str = None, kvdb_id: list = None) -> ConnexionResponse:
+    """List or get KVDBs.
+
+    Parameters
+    ----------
+    pretty : bool
+        Show results in human-readable format.
+    wait_for_complete : bool
+        Disable timeout response.
+    offset : int
+        First element to return in the collection.
+    limit : int
+        Maximum number of elements to return.
+    select : list
+        Select which fields to return (separated by comma).
+    sort : str
+        Sort the collection by a field or fields (separated by comma). Use +/- at the beginning
+        to list in ascending or descending order.
+    search : str
+        Look for elements with the specified string.
+    q : str
+        Query to filter results by.
+    distinct : bool
+        Look for distinct values.
+    type_ : str
+        Policy type. Allowed values: 'testing' | 'production'.
+    kvdb_id : list[str]
+        Filter by KVDB IDs (e.g. kvdb_id=foo&kvdb_id=bar).
+
+    Returns
+    -------
+    ConnexionResponse
+        API response.
+    """
+    f_kwargs = {
+        'policy_type': type_,
+        'ids': kvdb_id if kvdb_id else None,
+        'offset': offset,
+        'limit': limit,
+        'select': select,
+        'sort_by': parse_api_param(sort, 'sort')['fields'] if sort is not None else ['id'],
+        'sort_ascending': True if sort is None or parse_api_param(sort, 'sort')['order'] == 'asc' else False,
+        'search_text': parse_api_param(search, 'search')['value'] if search is not None else None,
+        'complementary_search': parse_api_param(search, 'search')['negation'] if search is not None else None,
+        'search_in_fields': ['id', 'name', 'integration_id'],
+        'q': q,
+        'distinct': distinct
+    }
+
+    dapi = DistributedAPI(
+        f=kvdbs.list_kvdbs,
+        f_kwargs=remove_nones_to_dict(f_kwargs),
+        request_type='local_master',
+        is_async=True,
+        wait_for_complete=wait_for_complete,
+        logger=logger,
+        rbac_permissions=request.context['token_info']['rbac_policies']
+    )
+    data = raise_if_exc(await dapi.distribute_function())
+
+    return json_response(data, pretty=pretty)
+
+
+async def post_kvdbs(body: dict, pretty: bool = False, wait_for_complete: bool = False,
+                     type_: str = None) -> ConnexionResponse:
+    """Create a KVDB (master node only).
```

Comment:
All operations are `local_master` so there's no point in highlighting this. Also, we don't usually do it in other controllers

```suggestion
    """Create a KVDB.
```